optimum/intel/openvino/modeling_decoder.py

```diff
@@ -301,15 +308,14 @@ def forward(
             inputs["attention_mask"] = np.array(attention_mask)
 
         # Run inference
-        outputs = self.request(inputs, shared_memory=True)
+        self.request.start_async(inputs, shared_memory=True)
```

Comment:
Is there any danger in using shared_memory in combination with async in this case?

Reply:
it is not truly the async since we wait for the results right after the call, it is more a way to access the results without copying them (this is what you get using the current synchronous API).

Reply:
Ah, thanks! This is a great solution for now. I heard sync inference in OpenVINO should be improved in the next release, with an option to not return the data.